tools/generate-contributors.ts

```diff
@@ -1,6 +1,7 @@
 // this script uses the github api to fetch a list of contributors
 // https://developer.github.com/v3/repos/#list-contributors
 // this endpoint returns a list of contributors sorted by number of contributions
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Actually, I just realized this probably isn't necessary; `@types/isomorphic-fetch` is missing...

Reply:
TIL: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/28611

Reply:
This'll be unnecessary once #3281 goes in. I'll hold off retargeting this PR or bringing those changes in here until that PR's strategy is confirmed.